test/torch/pointers/test_pointer_tensor.py

```diff
@@ -323,10 +323,11 @@ def test_move(workers):
 
     alice.clear_objects()
     bob.clear_objects()
-    x = torch.tensor([1.0, 2, 3, 4, 5]).send(bob)
+    t = torch.tensor([1.0, 2, 3, 4, 5])
+    x = t.send(bob)
```

Comment:
Can't we do `torch.tensor([1.0, 2, 3, 4, 5]).send(bob)` anymore?

Reply:
You can but I need to keep a ref to `t` :)

Reply:
Oops, indeed 👍